src/screens/home/home.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { StyleSheet } from 'react-native';
+
+import { step, white } from '../../config/styles';
+
+export const HomeStyles = StyleSheet.create({
+  accountInfo: {
+    display: 'flex',
+    flexDirection: 'column',
```

Comment:
You could omit `display: flex` & `flexDirection: column` as they are default values in RN